packages/components/src/components/contract-card/contract-card-items/turbos-card-body.tsx

```diff
@@ -8,6 +7,45 @@ import Icon from '../../icon';
 import MobileWrapper from '../../mobile-wrapper';
 import Money from '../../money';
 import { ResultStatusIcon } from '../result-overlay/result-overlay.jsx';
+import { TContractInfo, TGetContractUpdateConfig } from '@deriv/shared/src/utils/contract/contract-types';
+
+type TToastConfig = {
+    key?: string;
+    content: string;
+    timeout?: number;
+    is_bottom?: boolean;
+    type: string;
+};
+
+type TCardLabels = {
+    BARRIER_LEVEL: string;
+    BUY_PRICE: string;
+    CURRENT_PRICE: string;
+    STAKE: string;
+    TAKE_PROFIT: string;
+    TOTAL_PROFIT_LOSS: string;
+    PAYOUT: string;
+    PROFIT_LOSS: string;
+    POTENTIAL_PROFIT_LOSS: string;
+};
+type TTurbosCardBody = {
+    addToast: (toast_config: TToastConfig) => void;
+    connectWithContractUpdate?: any;
```

Comment:
`connectWithContractUpdate` is a function, that returns store connected wrapper for contract update form, so temporary its type will be `any`.

Reply:
connectWithContractUpdate: <T extends object>(Component: React.ComponentType<T>) => React.ComponentType<T>;